### ISSUE TITLE
Build failure: Missing string information for rutherford-scattering a11y.screenSummar

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rutherford-scattering#190]
Found on CT, this sim cannot build because of this error:

> Missing string information for rutherford-scattering a11y.screenSummar

This is actually https://github.com/phetsims/chipper/issues/1661, which came up during a recent dev meeting. This dynamic indexing confuses the scanner looking for used strings:

```ts
    super( {
      playAreaContent: RutherfordScatteringFluent.a11y.screenSummary[ screenType ].playAreaStringProperty,
      controlAreaContent: RutherfordScatteringFluent.a11y.screenSummary[ screenType ].controlAreaStringProperty,
      interactionHintContent: RutherfordScatteringFluent.a11y.screenSummary[ screenType ].interactionHintStringProperty
    } );
```

Could this be implemented a different way, maybe`RSScreenSummaryContent` could take a config with static string references?

**Comment by marlitas** [comment]
This has been fixed in a different issue. Closing.
